utils/helpers.ts

```
@@ -107,39 +146,96 @@ export function formatDuration(seconds: number): string {
   return parts.length > 0 ? parts.join(' ') : '0m';
 }
 
+/**
+ * Calculates the percentage change between two values.
+ * @param {number} currentValue - The current value
+ * @param {number} pastValue - The past/previous value for comparison
+ * @returns {number | 'N/A'} The percentage change, or 'N/A' if pastValue is zero (to avoid division by zero)
+ */
 export function calculatePercentageChange(currentValue: number, pastValue: number): number | 'N/A' {
```

ESLint

[error] Replace `currentValue:·number,·pastValue:·number` with `⏎··currentValue:·number,⏎··pastValue:·number⏎`

(prettier/prettier)